website/docs/cloud-docs/registry/publish-providers.mdx

```diff
@@ -20,72 +20,196 @@ Users must be members of an organization to access its registry and private prov
-
-Use the [Create a Provider endpoint](/terraform/cloud-docs/api-docs/private-registry/providers#create-a-provider) to create the provider in Terraform Cloud.
-
-The provider is available in your organization’s Terraform Cloud private registry, but consumers cannot use it until you add a version and a platform.
-
-### Create a Version and Platform
-
-To create a version, a platform, and upload the required release files:
-
-1. Confirm that you can access the provider binaries, public GPG signing key, `SHA256SUMS` file, and `SHA256SUMS.sig` file from at least one release.
-
-1. Use the [Add a GPG key endpoint](/terraform/cloud-docs/api-docs/private-registry/gpg-keys#add-a-gpg-key) to add the public key that matches the signing key for the release. The response contains a `key-id` that you will use to create a provider version.
-
-    ```json
-    "key-id": "32966F3FB5AC1129"
-    ```
+## Publishing a provider
 
-1. Use the [Create a Provider Version endpoint](/terraform/cloud-docs/api-docs/private-registry/provider-versions-platforms#create-a-provider-version) to create a version with the `key-id` from the previous step. The response includes URL links that you will use to upload the `SHA256SUMS` and `SHA256.sig` files.
-
-    ```json
-    "links": {
-        "shasums-upload":     "https://archivist.terraform.io/v1/object/dmF1b64hd73ghd63",
-        "shasums-sig-upload": "https://archivist.terraform.io/v1/object/dmF1b37dj37dh33d"
-      }
-    ```
-
-1. Upload the `SHA256SUMS` and `SHA256SUMS.sig` files to the URLs returned in the previous step. The example command below uploads the files from a local machine.
-
-    ```
-    $ curl -T local-example/terraform-provider-random_3.1.3_SHA256SUMS
-      https://archivist.terraform.io/v1/object/dmF1b64hd73ghd63
-    ```
-
-1. Use the [Create a Provider Platform endpoint](/terraform/cloud-docs/api-docs/private-registry/provider-versions-platforms#create-a-provider-platform) to create a platform for the version. Platforms are binaries that allow the provider to run on a particular operating system and architecture combination (e.g., Linux and AMD64). The response includes a `provider-binary-upload` URL that you will use to upload the binary file for the platform.
-
-    ```json
-    "links": {
-          "provider-binary-upload": "https://archivist.terraform.io/v1/object/dmF1b45c367djh45nj78"
-        }
-    ```
-
-1. Upload the platform binary file to the URL from the previous step. The example command below uploads the binary from a local machine.
-
-    ```
-    $ curl -T local-example/terraform-provider-random_3.1.3_linux_amd64.zip
-      https://archivist.terraform.io/v1/object/dmF1b45c367djh45nj78
-    ```
-
-1. (Optional) Repeat steps 4 and 5 to add additional platform binaries for the release.
+Before consumers can use a private provider, you must create the provider, upload a GPG signing key, create at least one version, create at least one platform for that version, and upload release files.
```

Comment:
Since you have to do a long list of items in sequence can we make this an ordered list?
```suggestion
Before consumers can use a private provider, you must do the following:
1. [Create the provider](#create-the-provider)
2. [Upload a GPG signing key](#add-your-public-key)
3. [Create at least one version](#create-a-version)
4. [Create at least one platform for that version](#create-a-provider-platform)
5. [Upload release files](#upload-provider-binary)
```